frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/utilization-card/multi-utilization-item.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { DataPoint } from '@console/internal/components/graphs';
+import { Humanize } from '@console/internal/components/utils';
+import { AreaChart } from './area-chart';
+
+export const MultilineUtilizationItem: React.FC<MultilineUtilizationItemProps> = React.memo(
+  ({ title, data, humanizeValue, isLoading = false, queries, error, chartType }) => {
+    const { t } = useTranslation();
+    const currentValue =
+      data.length > 1 && data[0].length && data[1].length
+        ? humanizeValue(data[0][data[0].length - 1].y + data[1][data[1].length - 1].y).string
+        : '';
+    const chart = (
```

Comment:
Instead of  passing `chartType` pass a chartComponent. You define the interface of the chart component. These chart components will have their own data handlers encapsulated with them.